App/Redux/TextileRedux.js

```diff
@@ -53,30 +56,17 @@ export const onboardedSuccess = state => {
   return state.merge({ onboarded: true })
 }
 
-export const handleImageAdded = (state, {image, remotePayloadPath}) => {
-  const existingItems = state.images.items ? state.images.items : []
-  const items = [{ image, remotePayloadPath, state: 'pending' }, ...existingItems]
-  return state.merge({ images: { items } })
-}
-
-export const handleImageSubmittedForUpload = (state, {path, hash, id}) => {
```

Comment:
Set the custom upload id to the hash, so we no longer need the reducer that persists the id.